common/src/main/java/org/jboss/hal/testsuite/page/config/ELResolverPage.java

```diff
@@ -0,0 +1,33 @@
+package org.jboss.hal.testsuite.page.config;
+
+import org.jboss.arquillian.graphene.Graphene;
+import org.jboss.arquillian.graphene.findby.ByJQuery;
+import org.jboss.hal.testsuite.fragment.ConfigFragment;
+import org.jboss.hal.testsuite.page.home.HomePage;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+/**
+ * Created by pcyprian on 26.10.15.
+ */
+public class ELResolverPage extends HomePage {
+
+    public void openExpressionResolver() {
+        WebElement tools = browser.findElement(ByJQuery.selector("div.gwt-HTML.footer-link:contains(Tools)"));
+        tools.click();
+        browser.findElement(By.xpath("//div[@class='popupContent']//a[contains(text(), 'Expression Resolver')]")).click();
+    }
+
+    public ConfigFragment getWindowFragment() {
```

Comment:
It seems that this method is now called just form `resolveSystemProperty(String name)` method of the same object thus I'm suggesting to either 

- make `getWindowFragment()` private or 
- get rid of `getWindowFragment()` method and move it's code inside of `resolveSystemProperty(String name)` method.